script/reference-device/dhcpcd.exit-hook

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#dhcpcd.exit-hook
+
+
+WAN_INTERFACE="eth0"
+WPAN_INTERFACE="wpan0"
+
+RADVD_CONF="/etc/radvd.conf"
```

Comment:
As far as I can remember, 2 issues we noticed today regarding radvd:

- Radvd is not installed.
- Radvd may not have a config file at this point.

Reply:
need handle permission to write files on the system, maybe change to root

Reply:
failure of radvd is also from mismatched OS release signature